### PR TITLE
Add project-assigner workflow

### PR DESCRIPTION
**Pull request by brokensound77**
## Issues
part of elastic/security-team/issues/1151

## Summary
Adds workflow to add all rule issues and PRs to the org project board

### CODE REVIEW

File: .github/workflows/project-assigner.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  issues:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  assign_to_project:
+    runs-on: ubuntu-latest
+    name: Assign or remove an issue from a project based on a label
+    steps:
```
Comment (brokensound77) [inline .github/workflows/project-assigner.yml:17]: Probably should add an `if` here to only run if it is not already added to the project, in order to not overwrite the status (unless it errors out on its own)

File: .github/workflows/project-assigner.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: Project-assigner
+
+on:
+  pull_request_target:
+    types:
+      - labeled
```
Comment (rw-access) [inline .github/workflows/project-assigner.yml:6]:
```suggestion
      - opened
      - labeled
```

**Review by rw-access — APPROVED** [review]
LGTM if it works

**Comment by brokensound77** [comment]
Starting to explore moving to the new beta projects. Will take a revised approach to workflow automation: https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects

closing

**Pull request closed without merging**